docs/UserGuide.md

```diff
@@ -302,6 +302,20 @@ Refer to [Finding a meeting](#finding-a-meeting-find) above for details of the `
 
 #### Finding a meeting: `find`
```

Comment:
Can consider using "Finding meetings" so that phrasing is consistent with contacts